ner_v2/detectors/numeral/number/number_detection.py

```diff
@@ -134,7 +151,7 @@ def detect_entity(self, text, **kwargs):
 
         """
         self.text = ' ' + text.lower() + ' '
```

Comment:
Hey for my learning, why do we add extra spaces to start and end of the string before passing to re.sub ?

Reply:
No concerete idea since this code was written years ago. This preprocessing is present throughout the code.
My best guess is to make boundaries more strict and text padding also enhances visibility.